content/python/concepts/built-in-functions/terms/setattr/setattr.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Setattr()'
+Description: 'Sets the value of the attribute of an object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Objects'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`Setattr()`** function is a built-in Python function used to set the value of a named attribute of an object. It allows you to dynamically assign or modify attributes of an object at runtime.
+
+## Syntax
+
+```pseudo
+setattr(object, attribute, value)
+```
+
+The `setattr()` function requires three parameters:
+
+- `object`: an object.
+- `attribute`: attribute name that you want to set.
+- `value`: the value you want to give the `attribute`.
+
+## Example
+
+The following example will change the `value` of the `name` attribute from the `Person` object:
+
+```py
+class Person:
+  name = "Alex"
+  age = 30
+
+# Updating the name property
+setattr(Person, 'name', "John")
+
+# Retrieving the name property
+new_name = getattr(Person, 'name')
+print("My new name is " + new_name)
+
+# output: My new name is John
+```
+## Codebyte Example
+
+Suppose we have a `Person` class that represents a person with attributes such as name and age. We can use the `setattr()` function to dynamically set the value of an attribute.
+
```

Comment:
[Nitpick/Proofreading] According to [the style guide(https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#general-writing-tips), pronouns are discouraged. 
```suggestion
Suppose there is a `Person` class that represents a person with attributes such as name and age. The `setattr()` function can be used to dynamically set the value of an attribute.

```